### ISSUE TITLE
Could not find com.google.android.gms:play-services-maps:8.4.0.

### ISSUE THREAD

**Issue by AaronNGray** [googlemaps/android-samples#3]
Am I missing a library ? If so what library and where do I obtain it from ?

~~~
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugCompile'.
   > Could not find com.google.android.gms:play-services-maps:8.4.0.
     Searched in the following locations:
         https://jcenter.bintray.com/com/google/android/gms/play-services-maps/8
.4.0/play-services-maps-8.4.0.pom
         https://jcenter.bintray.com/com/google/android/gms/play-services-maps/8
.4.0/play-services-maps-8.4.0.jar
         file:/C:/Users/Aaron/AppData/Local/Android/sdk2/extras/android/m2reposi
tory/com/google/android/gms/play-services-maps/8.4.0/play-services-maps-8.4.0.po
m
         file:/C:/Users/Aaron/AppData/Local/Android/sdk2/extras/android/m2reposi
tory/com/google/android/gms/play-services-maps/8.4.0/play-services-maps-8.4.0.ja
r
         file:/C:/Users/Aaron/AppData/Local/Android/sdk2/extras/google/m2reposit
ory/com/google/android/gms/play-services-maps/8.4.0/play-services-maps-8.4.0.pom

         file:/C:/Users/Aaron/AppData/Local/Android/sdk2/extras/google/m2reposit
ory/com/google/android/gms/play-services-maps/8.4.0/play-services-maps-8.4.0.jar

     Required by:
         ApiDemos:app:unspecified
~~~

**Comment by 4xes** [comment]
Perhaps you've solved the problem.
For someone who see this problem in future.
You need update `Android Support Repository` in SDK Manager.

**Comment by SteeveO** [comment]
I had this problem like yesterday and nothing seemed to work: I had Android Support Repository and Google Play Services updated and everything... 

Turns out the environment variable ANDROID_HOME was pointing to the wrong folder looking for the android sdk one. To change that I had to set it right in the .profile file at the root of my MacBook. 

Here's the syntax to do so : export ANDROID_HOME='/Users/SteeveO/Library/Android/sdk'  

Hope it helps someone in the future!
